lib/manifest.ex

```diff
@@ -271,8 +271,8 @@ defmodule Manifest do
 
   defp rollback([], acc), do: {:ok, acc}
 
-  defp rollback([{operation, {rollback, identifier}} | rest], acc) do
-    case rollback.(identifier) do
+  defp rollback([{operation, {rollback, identifier, previous}} | rest], acc) do
+    case rollback.(identifier, previous) do
```

Comment:
Can you make this work with both an arity of 1 and 2? Most likely it'll have to be a try block.